src/ploomber/tasks/notebook.py

```diff
@@ -913,6 +940,14 @@ def run(self):
             ]
         )
 
+        if self.local_execution:
```

Comment:
I'm a bit confused about this logic. 

seems like adding the notebook's parent should be independent from `local_execution`?

Like regardless of its value we can just add `str(self.source.loc.parent.resolve())` to the env to ensure modules are importable

Reply:
I agree. But in the documentation of NotebookRunner, it is written:
> local_execution (bool, optional) – Change the working directory to be the parent of the notebook’s source. Defaults to False. This resembles the default behavior when running notebooks interactively via Jupyter notebook

And as `ScriptRunner` has the same documentation with the same parameters, that's why I used it. But I do agree, there is no need of `local_execution` in `ScriptRunner` and by default `env` should be used.

Do you want me to remove it?

Reply:
yeah, let's make this fix independent of the `local_execution` parameter